mne/__init__.py

```diff
@@ -47,7 +47,8 @@
                       convert_forward_solution, make_field_map,
                       make_forward_dipole)
 from .source_estimate import (read_source_estimate, MixedSourceEstimate,
-                              SourceEstimate, VolSourceEstimate, morph_data,
+                              SourceEstimate, VectorSourceEstimate,
```

Comment:
One thing just occurred to me -- it would be possible also to have `VectorVolSourceEstimate` eventually, right?